static/js/modules/game/articleRenderer.js

```diff
@@ -86,12 +81,17 @@ export class ArticleRenderer {
             });
 
             // Remove "/wiki/" from string
-            this.loadPageWrapper(linkEl.getAttribute("href").substring(6))
+            this.loadPage(linkEl.getAttribute("href").substring(6))
         }
     }
 }
 
-
+function setMargin(frame) {
+    const desktop_ht = document.getElementById("time-box") ? document.getElementById("time-box").offsetHeight : 225;
+    const mobile_ht = document.getElementById("time-box-mobile") ? document.getElementById("time-box-mobile").offsetHeight : 225;
+    const margin = Math.max(desktop_ht, mobile_ht, 225) + 25;
+    frame.style.marginBottom = margin + "px";
+}
```

Comment:
hmm, I don't like it relying on setting the margin by assuming these classes exist. Why not just do it in the callback (in the games)? Cause there's nothing that the article renderer has that is used to compute this margin?

Reply:
I mean that's the point of the conditional:/ Can definitely do it in the callback but I want to limit marathon and play.js to the actual game logic as much as possible. Plus this way there's no code duplication. What's the problem with the conditional?

Reply:
I agree with hardcoding it actually, just put it to something pretty big lol